app/views/contacts/_fields.slim

```diff
@@ -26,18 +26,19 @@
     .col-xs-12
       = f.input :primary_phone
 
-.row
-  .col-xs-12
-    #emails
-      = f.simple_fields_for :contact_emails do |e|
-        = render 'contacts/email_fields', f: e
-      .links
-        = link_to_add_association t_title(:add, ContactEmail), f, :contact_emails, partial: 'contacts/email_fields', class: 'btn btn-xs btn-default'
+- unless f.object.volunteer?
```

Comment:
ah true, and if I write it like this, then I can remove the` volunteer? `method in the contact model, right?